doctest.cabal

```diff
@@ -51,7 +51,7 @@ library
   build-depends:
       base          == 4.*
     , base-compat   >= 0.7.0
-    , ghc           >= 7.0 && < 8.2
+    , ghc           >= 7.0 && < 8.3
```

Comment:
I think `< 8.4` is strictly better here, as it allows to build `doctest` with the in-development version of GHC.